crates/ruma-events/src/room/language.rs

```diff
@@ -0,0 +1,75 @@
+//! Types for the [`m.room.language`] event.
+//!
+//! [`m.room.language`]: https://github.com/matrix-org/matrix-spec-proposals/pull/4334
+
+use ruma_macros::EventContent;
+use serde::{Deserialize, Serialize};
+
+use crate::EmptyStateKey;
+
+/// The content of an `m.room.language` event.
+///
+/// The room language is a [IETF BCP 47](https://developer.mozilla.org/en-US/docs/Glossary/BCP_47_language_tag) language code.
```

Comment:
I believe that it would be better if this was documented directly on the `language` field.  It would also be preferable to link to the actual spec: https://datatracker.ietf.org/doc/bcp47/.